wagtail/bin/wagtail.py

```diff
@@ -59,14 +59,24 @@ class CreateProject(Command):
     description = "Creates the directory structure for a new Wagtail project."
 
     def add_arguments(self, parser):
+        import wagtail
```

Comment:
I suppose we could also do something like `__file__.parent().parent()`, but I'm not sure that's a better approach in case we move this file around